docs/developers/node-tutorial.md

```diff
@@ -35,7 +35,7 @@ We need to generate a wallet.
 Run the following command:
 
 ```sh
-celestia light keys add developer --keyring-backend test
+./cel-key add <key_name> --keyring-backend test --node.type light
```

Comment:
`cel-key` is not built in previous steps.
```suggestion
make install cel-key
cel-key add <key_name> --keyring-backend test --node.type light
```

Reply:
cc @renaynay for more specific changes

Reply:
i see now this too in the other guide:

# dumps binary in current working directory, accessible via `./cel-key`
make cel-key
or

# installs binary in GOBIN path, accessible via `cel-key`
make install-key

ref: https://github.com/celestiaorg/docs/blob/main/docs/nodes/keys.md

Reply:
Yes